### ISSUE TITLE
Arrows for foreign keys

### ISSUE THREAD

**Issue by matheusvellone** [PillFall/Figma-Database-Table#5]
Hi, thanks for this widget, just found it on Figma Community page!

I was wondering if there is a way to create a ForeignKey relation between two tables, with a custom arrow position, from the foreign field to the referenced one.

Don't know if Figma allows this, due to its 4 point location for arrows.

**Comment by PillFall** [comment]
I was thinking in that 😢. That is something I would love to have in the widget, but right now, as far as I could read in the Figma Widget documentation, there is no way to create custom anchor points for the FigJam connectors. So probably this is something that could not be added in a near future.

**Comment by matheusvellone** [comment]
Yeah, i've read some docs too, but could not find anything mentioning custom anchor points 😕 

Maybe they'll add support for this later. I did some search around the [forum](https://forum.figma.com/) but could not find anything useful, only [this thread](https://forum.figma.com/t/custom-objects-with-custom-anchor-connection-points/6793) but it was closed due to inactivity 😢

**Comment by sonnow** [comment]
This addition would be awesome! Does this announcement make the improvement possible?
https://forum.figma.com/t/launched-custom-connection-points-on-shapes-in-figjam/64614

**Comment by PillFall** [comment]
Great!

Seems like is a built-in feature, hope they let us put also more anchor point for a better UI
